<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMDActionSPDialog</class>
 <widget class="QDialog" name="CMDActionSPDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="protectModeComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>391</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>保护模式</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="recordSigResultCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>记录签名结果到日志</string>
   </property>
  </widget>
  <widget class="QLabel" name="sigTypeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>签名类型</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>481</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTabWidget" name="powerGroupTabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>500</width>
     <height>201</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="operatorGroupTab">
    <attribute name="title">
     <string>操作组</string>
    </attribute>
    <widget class="QListWidget" name="operatorGroupListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>380</width>
       <height>170</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="removeOperatorGroupButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>移除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifyOperatorGroupButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verifierGroupTab">
    <attribute name="title">
     <string>校验组</string>
    </attribute>
    <widget class="QListWidget" name="verifierGroupListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>380</width>
       <height>170</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="modifyVerifierGroupButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeVerifierGroupButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>移除</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>500</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="sigTypeComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>391</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
